app/models/unpublishing.rb

```diff
@@ -0,0 +1,17 @@
+class Unpublishing < ActiveRecord::Base
+  self.inheritance_column = nil
+
+  belongs_to :content_item
+
+  validates :type, presence: true
+  validates :explanation, presence: true, if: :withdrawal?
+  validates :alternative_url, presence: true, if: :redirect?
```

Comment:
Are there any cases where we'd have more than one unpublishing for the same content_item? If not, could we add a uniqueness validator for belt and braces?